_posts/release/2018-07-18-accumulo-1.9.3.md

```diff
@@ -14,6 +14,16 @@ Users of any previous version of 1.8 or 1.9 are encouraged to upgrade
 
 ## Notable Changes
 
+### Multiple Fixes for Write Ahead Logs
+
+Handle many tablets referencing many WALs [#854] [#860]
```

Comment:
These are a good start, but these should be expanded a bit to explain the changes to the user in a way that helps them understand how it impacts them. Several of these could issues could be listed under the same description, if the high-level overview is the same.